x-pack/legacy/plugins/canvas/shareable_runtime/api/index.ts

```diff
@@ -5,6 +5,7 @@
  */
 
 import 'whatwg-fetch';
-import 'babel-polyfill';
+import 'regenerator-runtime/runtime';
+import 'core-js/stable';
```

Comment:
core-js should almost certainly be the first thing we import, it handles a lot of low level polyfills that might be reused by regenerator and whatwg-fetch

Reply:
👍